utils/cautils/certificate_flow.go

```diff
@@ -35,16 +36,66 @@ type CertificateFlow struct {
 	offline   bool
 }
 
+type flowContext struct {
+	DisableCustomSANs       bool
+	SSHPublicKey            ssh.PublicKey
+	CertificateRequest      *x509.CertificateRequest
+	ConfirmationFingerprint string
+}
+
 // sharedContext is used to share information between commands.
-var sharedContext = struct {
-	DisableCustomSANs bool
-}{}
+var sharedContext flowContext
+
```

Comment:
Might be better to pass through the `cli.Context` instead of relying on this shared state, but I remember that might take some more refactoring.

Reply:
Unfortunately, the cli.Context doesn't allow adding custom attributes. We would need to extend the cli.Context with a new type and refactor several commands to do this.